src/G2/Translation/HaskellCheck.hs

```diff
@@ -29,6 +39,22 @@ validateStates proj src modN entry chAll gflags in_out = do
         loadToCheck proj src modN gflags
         mapM (runCheck modN entry chAll) in_out)
 
+-- Convert g2 generated types into readable string that aim to notify the environment about the types generated by g2
+g2GeneratedTypeToName :: (PrettyGuide, State t) -> (Name, AlgDataTy) -> ((PrettyGuide, State t), String)
+g2GeneratedTypeToName (pg, s) (x,y@(DataTyCon{data_cons = dcs, bound_ids = is})) =
+    -- "data " ++ show x ++ " " ++ show ids ++ " = " ++ show dcs
+    -- data maybe a = just a | nothing
+    --  data Name bound_ids = datacons 
```

Comment:
Cleanup

Reply:
Got it

Reply:
Do you think it's ok if I save the last comment and add it as part of the documentation because I believe it's a helpful guidance?